CODEOWNERS

```diff
@@ -18,110 +20,111 @@
-pkg/client @cilium/api
-pkg/completion/ @cilium/proxy
-pkg/components/ @cilium/agent
-pkg/controller @cilium/agent
-pkg/counter @cilium/bpf
-pkg/datapath @cilium/bpf
-pkg/datapath/ipcache/ @cilium/ipcache
-pkg/defaults @cilium/agent
-pkg/elf @cilium/bpf
-pkg/endpoint/ @cilium/endpoint
-pkg/endpointmanager/ @cilium/endpoint
-pkg/envoy/ @cilium/proxy
-pkg/fqdn/ @cilium/agent
-pkg/health/ @cilium/health
-pkg/hubble/ @cilium/hubble
-pkg/identity @cilium/policy
-pkg/ipcache/ @cilium/ipcache
-pkg/k8s/ @cilium/kubernetes
-pkg/k8s/apis/cilium.io/v2/ @cilium/api
-pkg/k8s/client/clientset/versioned/ @cilium/api
-pkg/k8s/client/informers/ @cilium/api
-pkg/kafka/ @cilium/proxy
-pkg/kvstore/ @cilium/kvstore
-pkg/labels @cilium/policy @cilium/api
-pkg/launcher @pkg/agent
-pkg/loadbalancer @cilium/loadbalancer
-pkg/lock @pkg/agent
-pkg/logging/ @cilium/cli
-pkg/mac @cilium/bpf
-pkg/maps/ @cilium/bpf
-pkg/metrics @cilium/metrics
-pkg/monitor @cilium/monitor
-pkg/monitor/api @cilium/api
-pkg/monitor/datapath_debug.go @cilium/bpf
-pkg/monitor/format @cilium/cli
-pkg/monitor/payload @cilium/api
-pkg/mountinfo @cilium/bpf
-pkg/mtu @cilium/bpf
-pkg/option @cilium/option
-pkg/pidfile @cilium/agent
-pkg/policy @cilium/policy
-pkg/policy/api/ @cilium/api
-pkg/proxy/ @cilium/proxy
-pkg/proxy/accesslog @cilium/api
-pkg/serializer @cilium/agent
-pkg/service @cilium/policy
-pkg/sysctl @cilium/bpf
-pkg/tuple @cilium/bpf
-plugins/cilium-cni/ @cilium/kubernetes
-plugins/cilium-docker/ @cilium/docker
-proxylib/ @cilium/proxy
-README.rst @cilium/docs
-test/ @cilium/ci
-tests/ @cilium/ci
+/operator/ @cilium/operator
+/pkg/annotation @cilium/kubernetes
+/pkg/api/ @cilium/api
+/pkg/bpf/ @cilium/bpf
+/pkg/byteorder/ @cilium/bpf @cilium/api
+/pkg/client @cilium/api
+/pkg/completion/ @cilium/proxy
+/pkg/components/ @cilium/agent
+/pkg/controller @cilium/agent
+/pkg/counter @cilium/bpf
+/pkg/datapath @cilium/bpf
+/pkg/datapath/ipcache/ @cilium/ipcache
+/pkg/defaults @cilium/agent
+/pkg/elf @cilium/bpf
+/pkg/endpoint/ @cilium/endpoint
+/pkg/endpointmanager/ @cilium/endpoint
+/pkg/envoy/ @cilium/proxy
+/pkg/fqdn/ @cilium/agent
+/pkg/health/ @cilium/health
+/pkg/hubble/ @cilium/hubble
+/pkg/identity @cilium/policy
+/pkg/ipcache/ @cilium/ipcache
+/pkg/ipam/ @cilium/ipam
+/pkg/k8s/ @cilium/kubernetes
+/pkg/k8s/apis/cilium.io/v2/ @cilium/api
+/pkg/k8s/client/clientset/versioned/ @cilium/api
+/pkg/k8s/client/informers/ @cilium/api
+/pkg/kafka/ @cilium/proxy
+/pkg/kvstore/ @cilium/kvstore
+/pkg/labels @cilium/policy @cilium/api
+/pkg/launcher @pkg/agent
+/pkg/loadbalancer @cilium/loadbalancer
+/pkg/lock @pkg/agent
+/pkg/logging/ @cilium/cli
+/pkg/mac @cilium/bpf
+/pkg/maps/ @cilium/bpf
+/pkg/metrics @cilium/metrics
+/pkg/monitor @cilium/monitor
+/pkg/monitor/api @cilium/api
+/pkg/monitor/datapath_debug.go @cilium/bpf
+/pkg/monitor/format @cilium/cli
+/pkg/monitor/payload @cilium/api
+/pkg/mountinfo @cilium/bpf
+/pkg/mtu @cilium/bpf
+/pkg/option @cilium/option
```

Comment:
Huh, I'm not sure I was aware this is a dedicated team. Should this be @cilium/cli ? Is there a meaningful distinction in team responsibilities / makeup?